doc/environment.md

```diff
@@ -0,0 +1,24 @@
+# Environment Variables Reference
```

Comment:
I think this documentation now needs love ❤️ - I'm trying to run this locally but some of these were renamed for some reason (as mentioned in another comment) and other variables seem new and it's not clear what's the correct value.

Reply:
Yep, I was getting a bit tired towards the end of this PR and didn't keep the docs up-to-date - I'll fix them up.